ocw_import/api.py

```diff
@@ -119,16 +119,15 @@ def convert_data_to_content(filepath, data, website):  # pylint:disable=too-many
     parent = None
     if len(s3_content_parts) >= 1:
         content_json = yaml.load(s3_content_parts[0], Loader=yaml.SafeLoader)
-        layout = content_json.get("layout", None)
         uid = content_json.get("uid", None)
         dirpath, filename = get_dirpath_and_filename(
             filepath, expect_file_extension=True
         )
         parent_uid = content_json.get("parent_uid", None)
-        if layout in COURSE_PAGE_LAYOUTS:
+        if dirpath == "content/pages":
```

Comment:
This doesn't work for nested pages, for example https://ocwnext.odl.mit.edu/courses/18-06sc-linear-algebra-fall-2011/pages/ax-b-and-the-four-subspaces/an-overview-of-key-ideas/

If we are assigning each resource to `CONTENT_TYPE_PAGE` in the else clause anyway, can we just remove this part of the if clause?